apps/condo/domains/resident/tasks/sendResidentMessage.task.js

```diff
@@ -0,0 +1,116 @@
+const { isArray, isEmpty, compact, get, omit, uniq } = require('lodash')
+const { v4: uuid } = require('uuid')
+
+const conf = require('@open-condo/config')
+const { getLogger } = require('@open-condo/keystone/logging')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { createTask } = require('@open-condo/keystone/tasks')
+
+const { COUNTRIES } = require('@condo/domains/common/constants/countries')
+const { MESSAGE_META } = require('@condo/domains/notification/constants/constants')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const {
+    fillDataByMessageTypeMeta,
+    renderTemplateString, hydrateItems,
+} = require('@condo/domains/resident/tasks/helpers/messageTools')
+const { Resident } = require('@condo/domains/resident/utils/serverSchema')
+
+const CHUNK_SIZE = 20
+const logger = getLogger('sendResidentMessage')
+
+/**
+ * Please have a look at SendResidentMessageService for json structure
+ * @param json
+ * @returns {Promise<{attemptsCount: (*), successCount: number}|{attemptsCount: number, successCount: number, json, error: Error}>}
+ */
+const sendResidentMessage = async (json) => {
+    const taskId = this.id || uuid()
+    let data
+
+    try {
+        data = JSON.parse(json)
+    } catch (error) {
+        logger.error({ msg: 'Error parsing JSON.', error: error.message, taskId })
```

Comment:
For better parsing key of the error message should be named as `err`
```suggestion
        logger.error({ msg: 'Error parsing JSON.', err: error.message, taskId })
```